docs/source/development.rst

```diff
@@ -63,3 +63,17 @@ Make a new version of Birdy in the following steps:
 See the bumpversion_ documentation for details.
 
 .. _bumpversion: https://pypi.org/project/bumpversion/
+
+Release a new version
+=====================
+
+Leveraging GitHub Workflows, maintainers can release new versions of Birdy automatically:
+
+* Within the repository secrets, set two API tokens (`PYPI_API_TOKEN` and `TEST_PYPI_API_TOKEN`) with push access to Birdy on PyPI and TestPyPI, respectively.
```

Comment:
I'd remove that since it does not have to be done again for the next releases.

Reply:
Got it. My API Tokens will remain valid indefinitely, but if ever they are revoked, the action will let us know.